tests/components/mqtt/test_sensor.py

```diff
@@ -582,6 +582,7 @@ async def test_entity_device_info_with_hub(hass, mqtt_mock):
         identifiers={("mqtt", "hub-id")},
         manufacturer="manufacturer",
         model="hub",
+        suggested_area="default_area",
```

Comment:
```suggestion
```

Remove this. Instead, add it to `DEFAULT_CONFIG_DEVICE_INFO_ID` and `DEFAULT_CONFIG_DEVICE_INFO_MAC`, and update test helpers `help_test_entity_device_info_with_identifier` + `help_test_entity_device_info_with_connection` to make sure it works.

Reply:
Thanks for the suggestion, the tests were adjusted accordingly.